### ISSUE TITLE
[FR: storage-resize-images] Add ability to resize existing images 

### ISSUE THREAD

**Issue by joehan** [firebase/extensions#103]
### [REQUIRED] Step 2: Extension name

`storage-resize-images`

### What feature would you like to see?

FR from CLI UXRR - After reconfiguring the size on this extension, the participant wanted a way to resize all the old images that users have uploaded. Could we add a http endpoint that takes a path to an existing image in storage and resizes it.

### How would you use it?

Imagine this scenario: I use this extension to resize profile pics for my users. I redesign my website and want a slightly different size of profile pic, so I reconfigure this extension to my new size. With the FR, i could also easily go back and get the new size that i need for all my existing users.

**Comment by karayu** [comment]
Thanks Joe! We chatted a bit about this offline and it sounds like the actual ask here is to be able to backfill data after first install. This isn't the first extension to have this request. The firestore-bigquery-export extension does a one-time export of data before doing streaming updates. 

We'd like to roll this into a longer term more generalized workstream of allowing for initialization script when installing an extension, including one to backfill existing data/asset. 

Closing this for now but we'll be tracking internally. Thanks so much for filing!
